packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardEmail.stories.js

```diff
@@ -17,20 +17,14 @@
  */
 
 import lng from '@lightningjs/core';
-import mdx from './Keyboard.mdx';
-import { CATEGORIES } from '../../docs/constants';
 import { context } from '../../globals';
 import utils from '../../utils';
 import { default as KeyboardEmailComponent } from './KeyboardEmail';
 import { Keyboard } from './Keyboard.stories';
 
 export default {
-  title: `${CATEGORIES[8]}/KeyboardEmail`,
-  parameters: {
-    docs: {
-      page: mdx
-    }
-  }
+  title: 'Components/KeyboardEmail',
+  docs: { autodocs: false }
```

Comment:
fine for now, but wondering if in the future we should find a way to redirect to the base Keyboard docs to make it clear that this extends that?

Reply:
I'll add a follow up ticket in Jira for documentation that needs to be linked to multiple stories